### ISSUE TITLE
Implement html lang tag

### ISSUE THREAD

**Issue by alia-peterson** [CourseChart/course-chart-fe#23]
**Describe the feature you'd like**
Browser language should be specified for accessibility. 

Tried adding usual `<html lang='en'></html>` tag inside of Layout component but an error occurred that said that a div can't be inside of an html (or something like that). It seems like this is a Next.js specific issue that may be solved by some of the steps discussed [here](https://medium.com/weekly-webtips/accessibility-and-seo-optimizations-in-next-js-app-472515a50ffc). I was nervous to implement a `_document` file because I don't know how it will interact with out Layout file. Does it replace the Layout? Or can we have both with no issues?
